alignak/objects/item.py

```diff
@@ -634,85 +637,6 @@ def del_comment(self, comment_id):
         if c_to_del is not None:
             self.comments.remove(c_to_del)
 
```

Comment:
Moved in alignak/objects/schedulingitem.py because only used by object inherite schedulingitem